### PR TITLE
INT-4: Develop for beta

### PR DESCRIPTION
**Pull request by fernandacanto**
Adding the develop changes in beta that was created from the master.
This merge is necessary for us to validate that the master and develop changes are in beta.

Please do not merge without talking to me and @thiagosaife .

### CODE REVIEW

File: src/directives/tooltip/__tests__/VTooltip.spec.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
+import { mount } from '@vue/test-utils'
+
+// import { mountAttachingComponent } from '../../../utils/tests-utils'
```
Comment (thiagosaife) [inline src/directives/tooltip/__tests__/VTooltip.spec.js:3]: @fernandacanto Could you remove this commented line here, please?

**Review by thiagosaife — CHANGES_REQUESTED** [review]
Hi Fernanda. Great job. I've just requested two comment lines to be removed and I suggest that you run the unit tests before the merge. Congratulations, so far.

File: src/components/Button/__tests__/Button.spec.js [hunk 2]
```diff
@@ -143,8 +155,12 @@ describe('Test SbButton Component', () => {
       },
     })
 
-    it('should have an Tooltip component with correct text', () => {
-      expect(wrapper.find('.sb-tooltip').text()).toBe(iconDescription)
+    it('should have a tooltip with correct text', async () => {
+      await wrapper.find('button').trigger('focus')
+      // expect(wrapper.find('.sb-tooltip').text()).toBe(iconDescription)
```
[the remark this answers is not in the document]
Comment (fernandacanto) [reply src/components/Button/__tests__/Button.spec.js:160]: ok @thiagosaife !

File: src/directives/tooltip/__tests__/VTooltip.spec.js (hunk 1 above)
Comment (fernandacanto) [reply src/directives/tooltip/__tests__/VTooltip.spec.js:3]: Sure!

**Pull request merged**